api/v1/flow/flow.proto

```diff
@@ -472,3 +472,91 @@ message LostEvent {
     // PERF_EVENT_RING_BUFFER.
     google.protobuf.Int32Value cpu = 3;
 }
+
+// AgentEventType is the type of agent event. These values are shared with type
+// AgentNotification in pkg/monitor/api/types.go
+enum AgentEventType {
+    UNSPECIFIED = 0;
```

Comment:
I know this has been copied over verbatim from pkg/monitor/api/types.go, but because protobuf enum variants are unfortunately not namespaced, please use `AGENT_EVENT_UNKNOWN` (or similar) here. 

We have historically not been consistent with the default enum variant, but I've been trying to establish the `<TYPE>_UNKNOWN` naming for newly added enums.

Reply:
That answers my questions from https://github.com/cilium/cilium/pull/14168#discussion_r530385060, thanks. Will adjust to `AGENT_EVENT_UNKNOWN`.